c2corg_ui/static/js/maincontroller.js

```diff
@@ -69,3 +69,17 @@ app.MainController.prototype.isPath = function(path) {
   }
 };
 
+/**
+ *
+ * @export
+ */
+app.MainController.prototype.animateHeaderIcon = function(e) {
```

Comment:
This function should rather be moved to https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/utils.js